OutSystems/outsystems-maps
# ROU-4591: change how addresses are transformed into coordenates

This PR is for changing the way addresses are transformed into coordinates.

### What was happening
The transformation of the addresses, was using a google REST API, that implied making a fetch in the maps code. Additionally there were situations in which the translation of the addresses failed.

### What was done
* Change the code to start using the method `geo.geocode`, available out of the box in the Google Maps script.

### Test Steps
* Scenario 1
  * Open the screen
  * Click on the button Go to map
  * Click on the toggle above the map
    * First time the map will disappear
    * Second time the map will re-appear, but now running the new code
  * Start typing Rua de Santa Maria in the OutSystems Search Places
  * Notice that the map has changed the center of it

(The current code, **fails** to change the center of the map).


### Checklist
